### PR TITLE
Clarify the behavior of floating point operations in the presence of NaN

### PR DESCRIPTION
**Pull request by sunfishcode**
These could be assumed, but it's nice to make them explicit.

### CODE REVIEW

**Comment by jfbastien** [comment]
I think it would be useful to have a footnote on what "ordered" and "unordered" means. Otherwise lgtm.

**Comment by sunfishcode** [comment]
@jfbastien Done.

**Comment by titzer** [comment]
lgtm

File: AstSemantics.md [hunk 1]
```diff
@@ -465,18 +465,21 @@ implementations of the remaining required operations.
...
+  * `float64.gt`: compare ordered and greater than
+  * `float64.ge`: compare ordered and greater than or equal
   * `float64.sqrt`: square root
   * `float64.min`: minimum (binary operator); if either operand is NaN, returns NaN
   * `float64.max`: maximum (binary operator); if either operand is NaN, returns NaN
 
 `min` and `max` operations treat `-0.0` as being effectively less than `0.0`.
 
+In floating point comparisons, the operands are *unordered* if either operand
+is NaN, and *ordered*.
```
Comment (jfbastien) [inline AstSemantics.md]: and *ordered* if what? Seems like something is missing.

**Comment by sunfishcode** [comment]
@jfbastien ordered *otherwise*. Fixed now.

**Pull request merged**